mkosi/config.py

```diff
@@ -4597,6 +4604,7 @@ def summary(config: Config) -> str:
                          Passphrase: {none_to_none(config.passphrase)}
                            Checksum: {yes_no(config.checksum)}
                                Sign: {yes_no(config.sign)}
+                       OpenPGP Tool: ({config.openpgp_tool or "gpg"})
```

Comment:
You can drop the fallback here since the field will always be set as it has a default value